docs/content/envoy-external-authorization.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Envoy External Authorization
```

Comment:
Let's make the title `Envoy Authorization` instead. Looks a bit more consistent in the ToC.